chart/templates/deployment.yaml

```diff
@@ -38,8 +38,34 @@ spec:
             value: "{{ .Values.kafka.keystore }}"
           - name: JVM_OPTS
             value: "{{ .Values.jvm.opts }}"
+{{- if .Values.jmx.port }}
```

Comment:
Why do we need the `if` and `end` conditional blocks? Can't we just set the value, assuming there is a default in `values.yaml`?

Reply:
Threre is already default values in kafdrop application.yaml and docker entrypoint (kafdrop.sh), so i prefer not overriding them with some others/null values. It seems cleaner to me.
If i leave it as is, it's not so clean, but if i remove them, we need to synchronize default values  (values.yaml and application.yaml/kafdrop.sh)

What is your opinion ? Doesn’t matter for me ;)

Reply:
I would prefer to avoid conditional macros and just deal with defaults that are clearly stated in `values.yaml`, so that anyone who wants to know what an unspecified value will default to can check `values.yaml`.

Reply:
No problemo, i'm making changes.